iwyu_tool.py

```diff
@@ -203,6 +209,8 @@ def custom_help():
     parser.add_argument('source', nargs='*',
                         help='Zero or more source files to run IWYU on. '
                         'Defaults to all in compilation database.')
+    parser.add_argument('-i', '--ignore', nargs='+', default=[],
```

Comment:
Shouldn't  it be nargs='*'? I expected `+` to require at least one ignore-dir, which we don't want, right?